CoreFoundation/Base.subproj/CoreFoundation_Prefix.h

```diff
@@ -20,7 +20,11 @@
 
 #include <CoreFoundation/CFAvailability.h>
 
+#if TARGET_OS_WASI
```

Comment:
Huh; do things even build correctly without dispatch, or at least shims? I'm pretty sure we depend on `dispatch_*` kind of all over the place, which is why `__HAS_DISPATCH__` was kept fixed at 1.

Reply:
Yes, at least a certain subset (albeit a useful subset) does. That's why there's more `#if TARGET_OS_WASI` all over the codebase of our fork, more than a 100 files touched, and certain APIs are disabled completely. I'm going to submit those as separate PRs obviously.

Reply:
Is WASI strictly single-threaded?

Reply:
Yes, multi-threading support and atomics aren't standardised yet, and will be available in some future (unspecified) browser versions. Only Chrome has a working implementation hidden with experimental flags, as as I remember.